duckduckgo/macos-browser
# Bump privacy dashboard to latest version

Task/Issue URL: n/a
Tech Design URL: n/a
CC: @mallexxx 

**Description**:
Brings in support for new permissions model structure

**Steps to test this PR**:
1. Build browser from this branch
2. Go to https://permission.site/,select location permissions and allow
3. Open Privacy Dashboard — geolocation icon should show, browser shouldn’t crash when changing permissions status


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
